sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -258,21 +286,24 @@ async def create_database(
         offer_throughput: Optional[Union[int, ThroughputProperties]] = None,
         initial_headers: Optional[Dict[str, str]] = None,
         throughput_bucket: Optional[int] = None,
+        return_properties: bool = False,
         **kwargs: Any
-    ) -> DatabaseProxy:
+    ) -> Union[DatabaseProxy, tuple[DatabaseProxy, CosmosDict]]:
         """
         Create a new database with the given ID (name).
 
         :param str id: ID (name) of the database to create.
         :keyword offer_throughput: The provisioned throughput for this offer.
         :paramtype offer_throughput: Union[int, ~azure.cosmos.ThroughputProperties]
+        :keyword bool return_properties: Specifies function to return either a DatabaserProxy or a CosmosDict instance.
```

Comment:
Would this not be "Return a DatabaseProxy or a Tuple of DatabaseProxy and a CosmosDict"?